src/main/java/filter/expression/Qualifier.java

```diff
@@ -898,4 +969,53 @@ private void applyState(TurboIssue issue) throws QualifierApplicationException {
     public String getName() {
         return name;
     }
+
+    private Qualifier replaceMilestoneAliasQualifier(List<TurboMilestone> allMilestones, int currentIndex) {
+        String contents = content.get();
+        String alias = contents;
+
+        Pattern aliasPattern = Pattern.compile("(curr(?:ent)?)(?:(\\+|-)(\\d+))?");
+        Matcher aliasMatcher = aliasPattern.matcher(alias);
+        int offset = 0;
+
+        while (aliasMatcher.find()) {
+            if (aliasMatcher.group(2) != null && aliasMatcher.group(3) != null) {
+                offset = Integer.parseInt(aliasMatcher.group(3));
+                if (aliasMatcher.group(2).equals("+")) {
+                    currentIndex += offset;
+                } else {
+                    currentIndex -= offset;
+                }
+            } else if (!aliasMatcher.group(0).equalsIgnoreCase(alias)) {
```

Comment:
The regex could be changed to `current((\+|-)\d+)?$` (some details left out) to preclude this case and make `find` return false if it happens.